README.md

```diff
@@ -1,35 +1,41 @@
 # Shuttle Deploy Action
 
-This action automates the deployment of a rust project to [Shuttle](https://www.shuttle.rs/). This action builds the project and then deploys the result to Shuttle.
+This action automates the deployment of a Rust project to [Shuttle](https://www.shuttle.rs/). This action deploys the project to Shuttle, which builds it and hosts it.
 
 Note that you need to have created a project on Shuttle before you can deploy to it. This action will NOT create a new project for you.
 You can see the documentation on how to create a project [here](https://docs.shuttle.rs/introduction/quick-start).
 
+Shuttle Secrets are not handled by this action (yet). Add secrets using Secrets.toml with a manual deploy command. Read more about secrets [here](https://docs.shuttle.rs/resources/shuttle-secrets).
+
 ## Inputs
 
 | Name | Description | Required | Default |
 | --- | --- | --- | --- |
 | deploy-key | The Shuttle API key | true | N/A |
+| cargo-shuttle-version | Version of cargo-shuttle | false | `""` (latest) |
 | working-directory | The directory which includes the `Cargo.toml` | false | `"."` |
+| name | The directory which includes the `Cargo.toml` | false | `"."` |
 | allow-dirty | Allow uncommitted changes to be deployed | false | `"false"` |
+| no-test | Don't run tests before deployment | false | `"false"` |
 
 ## Outputs
 
 | Name | Description |
 | --- | --- |
-| shuttle-url | The URL of the deployed project |
+<!-- | shuttle-url | The URL of the deployed project | -->

```

Comment:
Should we remove this? It's currently an empty table, but I suppose the output is just the output from the `deploy` command, right? We could also rewrite it to reflect that.

Reply:
Log output and "outputs" are different things. The latter is like return variables https://docs.github.com/en/actions/using-jobs/defining-outputs-for-jobs , which we don't have any of yet, but i.e. the URL is very nice to add in the future, therefore commented.

Reply:
Oh, TIL! That makes sense.